### DISCUSSION TITLE
sea-orm-cli: auto-generated postgres enums aren't inserting the schema prefix to the enum name

### DISCUSSION THREAD

**Discussion by cbio-jimmy** [SeaQL/sea-orm#1658] [General]
## Description

when `sea-orm-cli generate entity -s schema_name` is used, the schema name isn't prefixing to the enum-name.

## Steps to Reproduce

1. Create an enum that uses an schema
```sql
create schema cards;

create type cards.genre as enum (
   'YUGIOH'
   'ONEPIECE'
);

create table cards.set (
    genre cards.genre,
);
```
2. Run `sea-orm-cli generate entity -s cards`

### Expected Behavior

I'm expecting it to look like
`#[sea_orm(rs_type = "String", db_type = "Enum", enum_name = "cards.genre")]`

### Actual Behavior

With the given schema name and enum, The generated code looks like
`#[sea_orm(rs_type = "String", db_type = "Enum", enum_name = "genre" `
It drops the schema name from the enum name

### Reproduces How Often

Always reproducible

### Workarounds

Manually adding the schema name in the generated code

## Versions

0.10.1+

**Comment by tyt2y3** [comment]
It'd be a good-to-have. PR would be welcomed.
